Runtime/Cesium3DTileset.cs

```diff
@@ -70,6 +70,16 @@ internal static void
             }
         }
 
+        public event Action<GameObject> OnNewGameObjectCreated;
```

Comment:
Maybe the name should be more specific to the tileset, like `OnTileGameObjectCreated`? @kring @csciguy8 do you have any strong opinions?